pkgs/applications/video/mpv/scripts/memo.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub
+, buildLua
+, lib
+, unstableGitUpdater
+}:
+
+buildLua {
+  pname = "memo";
+  version = "unstable-2023-10-31";
```

```suggestion
  version = "0-unstable-2023-10-31";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming